pkg/printers/internalversion/printers.go

```diff
@@ -715,7 +715,12 @@ func getServiceExternalIP(svc *api.Service, wide bool) string {
 	case api.ServiceTypeLoadBalancer:
 		lbIps := loadBalancerStatusStringer(svc.Status.LoadBalancer, wide)
 		if len(svc.Spec.ExternalIPs) > 0 {
-			result := append(strings.Split(lbIps, ","), svc.Spec.ExternalIPs...)
+			var result []string
```

Comment:
Would it read better like:
```
results := []string{}
if len(lbIPs) > 0 {
    results = append(results, strings.Split(lbIPs, ",")
}
results = append(results, svc.Spec.ExternalIPs)
```
Performance wise it might be a little slower with the allocations and appends but it would read/review quite easily.